src/classes/Recipe.js

```diff
@@ -8,20 +9,46 @@ class Recipe {
     this.instructions = recipe.instructions
     this.name = recipe.name
     this.tags = recipe.tags
- }
+  }
 
   getTotalCost() {
     let total = this.ingredients.reduce((acc, ingredient) => {
       return acc = acc += ingredient.estimatedCostInCents * ingredient.amount
     }, 0) / 100
     return `$${total.toFixed(2).toString()}`
   }
-  
+
   instantiateIngredients(recipe, ingredientsData) {
-    return recipe.ingredients.map(recipeObject => {
+    let a = recipe.ingredients.map(recipeObject => {
       let targetDatasetObject = ingredientsData.find(datasetObj => recipeObject.id === datasetObj.id)
       return new Ingredient(targetDatasetObject, recipeObject)
     })
+    return this.mergeIngredients(a)
+  }
+
+  mergeIngredients(ingredientsData) {
```

Comment:
This is a small thing, but maybe it'd be less confusing if the name of the `mergeIngredients()` parameter was something else, since "ingredientsData" is already the name of a parameter in `instantiateIngredients()`, which isn't the same thing as the arg being passed into `mergeIngredients()`?

Reply:
No problem! I will make this changes and push again.

Reply:
awesome!

Reply:
changing this param to `ingredientsDataWithDuplicates`